includes/documents/abstract-wcpdf-order-document-methods.php

```diff
@@ -674,32 +672,31 @@ public function get_tax_rate( $item, $order, $force_calculation = false ) {
 				}
 			}
 
-			$tax_rates = implode(', ', $tax_rates );
-			return $tax_rates;
+			return implode( ', ', $tax_rates );
 		}
 
-		if ( $line_tax == 0 ) {
+		if ( 0 === $line_tax ) {
```

Comment:
The `get_total_tax()` function returns a `string`, but maybe we should also consider that the value could be a `float`?

Reply:
I think we always have a `%` sign. In line `662` and also in `679`.

Reply:
@alexmigf Do you think I should consider `float` too?

Reply:
In the test I made the `$line_tax` is a `string`with a decimal value in there, no percentage there, so I think it's safer to consider also the `float`.